site/app/templates/forum/ForumBar.twig

```diff
@@ -61,15 +60,19 @@
 
 						{% endif %}
 					{% endfor %}
+
+					<span id="forum_filters">Filters: <span id="filter_names">None</span> &middot; <a onclick="clearForumFilter()" id="filter_close">Clear </a></span>
```

Comment:
this does not wrap well, especially if you have multiple options enabled (and I have no idea if we will be adding more options in the future

Reply:
@thef1stofhername I am not very good with frontend..hence would need your suggestion or pointer on how to make this better :(